### ISSUE TITLE
.rotateTowards() in an animation section being given a template is throwing an error

### ISSUE THREAD

**Issue by Vauryx** [fantasycalendar/FoundryVTT-Sequencer#28]
This effect is throwing an error only when given a template to rotateTowards() in the animation section
```javascript
async function finalTeleport(caster, location){
            let distance = Math.sqrt(Math.pow((location.x - caster.x), 2) + Math.pow((location.y - caster.y), 2));
            let steelWindSequence = new Sequence()
                    .effect()
                    .atLocation(caster)
                    .JB2A()
                    .file(gustAnim)
                    .reachTowards(location)
                    .opacity(0.8)
                    .fadeOut(250)
                    .belowTokens()
                    .animation()
                    .on(caster)
                    .rotateTowards(location)
                    .animation()
                    .on(caster)
                    .moveTowards(location, { ease: "easeOutElasticCustom" })
                    .moveSpeed(distance / 60)
                    .duration(800)
                await steelWindSequence.play();
        }
```
![image](https://user-images.githubusercontent.com/32877348/132436617-23487540-2d02-4f39-bbb3-1b8743530c69.png)

**Comment by Haxxer** [comment]
Could you provide a simple macro to recreate the error? :)

**Comment by Vauryx** [comment]
I should correct myself - The error happens specifically when given a templateDocument and not a template. 
You should be able to run the following macro with just a token selected. It will try to rotate the token to the template using its template document
```javascript
let caster = canvas.tokens.controlled[0];
let preReticle;
let reticle;

let templateData = {
    t: "circle",
    user: game.user._id,
    distance: 2.5,
    direction: 0,
    x: 0,
    y: 0,
    texture: ""
};

async function testRotation(templateDocument) {
    console.log(templateDocument);
    await new Sequence()
    .animation()
        .on(caster)
        .rotateTowards(templateDocument)
    .play()
}

async function createTemplate(reticleData) {
    preReticle = new MeasuredTemplateDocument(reticleData, { parent: canvas.scene });
    reticle = new game.dnd5e.canvas.AbilityTemplate(preReticle);
    reticle.actorSheet = caster.actor.data.document.sheet;
    Hooks.once("createMeasuredTemplate", async (template) => {
        await testRotation(template);
    });
    await reticle.drawPreview();
}

createTemplate(templateData);
```

https://user-images.githubusercontent.com/32877348/132582186-26b9f65a-337b-4c27-bf4b-0eb49887735c.mp4


![image](https://user-images.githubusercontent.com/32877348/132582016-fb57a679-c2db-4827-8bab-d016b4ecf688.png)

**Comment by Haxxer** [comment]
Resolved in 1.0.0
